docs(integrations): add etsy (#60)
myparcelnl/developer: src/nl/documentatie/30.etsy.md

# Etsy

## Inleiding

Creëer snel verzendetiketten van jouw Etsy-orders en download deze vervolgens allemaal tegelijk vanuit het MyParcel-platform.

## Installatie

### Selecteer de Etsy integratie

> Navigeer in de MyParcel backoffice naar Shopinstellingen > Integratie en scroll naar Etsy. Klik op de button [Geef toestemming]

Author: done

> Navigeer in de MyParcel backoffice naar [Shopinstellingen > Integratie](https://backoffice.myparcel.nl/settings/integration) en scroll naar Etsy. Klik op de button [Geef toestemming]

Reviewer: ```suggestion
Navigeer in de MyParcel Backoffice naar [Shopinstellingen > Integratie](https://backoffice.myparcel.nl/settings/integration) en scroll naar Etsy. Klik op de button "Geef toestemming"
```

> <MPImg src="/documentation/etsy/etsy_shopinstellingen_integratie.png" alt="Etsy integratie" />

Author: Done

> ### Etsy login

Reviewer: ```suggestion
### Inloggen bij Etsy
```

> Gebruik jouw Etsy login credentials om in te loggen en een connectie te maken tussen MyParcel en Etsy.

Reviewer: ```suggestion
Log in bij Etsy en maak een connectie te tussen MyParcel en jouw webwinkel.
```

<MPImg src="/documentation/etsy/etsy_login_credentials.png" alt="Etsy login" />

### Terugkoppeling van Etsy

> In de MyParcel backoffice wordt nu een terugkoppeling getoond van de Etsy koppeling

Reviewer: ```suggestion
In de MyParcel Backoffice wordt nu een terugkoppeling getoond dat het koppelen met Etsy geslaagd is.
```

<MPImg src="/documentation/etsy/etsy_koppeling_is_gemaakt.png" alt="Etsy koppeling" />

## Gebruik koppeling

### Importeren van orders

> Om je orders van Etsy te importeren, klik je in het zendingenoverzicht op en dan op Upload Etsy orders. Selecteer je meest recente Etsy besteloverzicht. De orders zullen ingeladen worden. Nadat je op hebt geklikt zullen de orders als concepten in je zendingenoverzicht komen te staan.

Reviewer: ```suggestion
Om je orders van Etsy te importeren, klik je in het zendingenoverzicht op "Importeer externe zendingen". Openstaande Etsy orders zullen ingeladen worden. Selecteer welke orders je wilt importeren en druk op "Importeren". Deze orders zullen als concepten in je zendingenoverzicht komen te staan.
```

> Je zendingen worden geïmporteerd als standaardzending. Welk pakkettype een standaardzending heeft, kun je instellen in je [backoffice bij > Shopinstellingen > Zendingen.](https://backoffice.myparcel.nl/settings/shipment)

Reviewer: ```suggestion
Je zendingen worden geïmporteerd als standaardzending. Welk pakkettype een standaardzending heeft, kun je instellen in je MyParcel Backoffice bij > [Shopinstellingen > Zendingen](https://backoffice.myparcel.nl/settings/shipment).
```

### Veelgestelde vragen over het gebruik van de plugin

#### Worden de barcodes teruggekoppeld naar Etsy?

> _Ja, de barcodes zullen teruggekoppeld worden aan Etsy en de bestelling zal de status "verzonden" krijgen zodra het label geprint is._

Reviewer: ```suggestion
_Ja, de barcodes worden teruggekoppeld aan Etsy en de bestelling zal de status "verzonden" krijgen zodra het label geprint is._
```
